nilearn/glm/thresholding.py

```diff
@@ -173,8 +172,9 @@ def cluster_level_inference(stat_img, mask_img=None,
     return proportion_true_discoveries_img
 
 
-def map_threshold(stat_img=None, mask_img=None, alpha=.001, threshold=3.,
-                  height_control='fpr', cluster_threshold=0, two_sided=True):
+def threshold_stats_img(stat_img=None, mask_img=None, alpha=.001, threshold=3.,
+                        height_control='fpr', cluster_threshold=0,
+                        two_sided=True):
```

Comment:
We should add a "see also" section to this function that points to nilearn.image.threshold_img, and vice versa.